evmos/assetlist.json

```diff
@@ -111,21 +111,21 @@
       }
     },
     {
-      "description": "The token of Teledisko DAO.",
+      "description": "The legacy token of Teledisko DAO.",
       "denom_units": [
         {
           "denom": "erc20/0x1cFc8f1FE8D5668BAFF2724547EcDbd6f013a280",
           "exponent": 0
         },
         {
-          "denom": "berlin",
+          "denom": "berlin-legacy",
           "exponent": 18
         }
       ],
       "base": "erc20/0x1cFc8f1FE8D5668BAFF2724547EcDbd6f013a280",
-      "name": "Teledisko DAO",
-      "display": "berlin",
-      "symbol": "BERLIN",
+      "name": "Teledisko DAO - Legacy",
+      "display": "berlin-legacy",
+      "symbol": "BERLIN-legacy",
```

Comment:
Thoughts on BERLIN.legacy instead?

Reply:
uhm would have been an option, but I'd have now to change again the metadata on chainlist... is this a deal breaker?

Reply:
I'm not too sure, there isn't a super strict standard at the moment.